index.js

```diff
@@ -1 +1,130 @@
+const popup = document.querySelector(".popup");
 
+const inputFormAddTask = document.querySelectorAll(".form__input-task");
+const textareaFormAddTask = document.querySelector(".form__input-note");
+const openPopupButton = document.querySelector(".add-task__button");
+const closePopupButton = document.querySelector(".popup__close");
+const formSubmitButton = document.querySelector(".form__submit-btn");
+const taskList = document.querySelector(".todo__list");
+const taskBody = document.querySelector(".task-card__template");
+let taskRemove = document.querySelectorAll(".todo-item .todo-item__remove-btn");
+const form = document.querySelector('#addTask');
+
+function disabledSubmitButton() {
+  formSubmitButton.setAttribute("disabled", "true");
+}
+
+function removeBtnActive() {
+  taskRemove = document.querySelectorAll(".todo-item .todo-item__remove-btn");
+  taskRemove.forEach(function (btn) {
+    btn.addEventListener('click', function () {
+      this.parentElement.remove()
+    });
+  });
+}
+
+function clearForm(form) {
+  form.querySelector(".form__input-task").value = "";
+  form.querySelector(".form__input-note").value = "";
+}
+
+// Open and Close popup
+function removePopup() {
```

Comment:
Действия которые осуществляются путём ручного нажатия/включения/выключения принятно именовать через префикс handle + название функции не совсем правильно передаёт её суть, лучше: 
```suggestion
function handleClosePopup() {
```